manager/controllers/app/blueprint_controller.go

```diff
@@ -93,10 +93,6 @@ func (r *BlueprintReconciler) reconcileFinalizers(blueprint *app.Blueprint) (ctr
 			// the finalizer is present - delete the allocated resources
 			if err := r.deleteExternalResources(blueprint); err != nil {
 				r.Log.V(0).Info("Error while deleting owned resources: " + err.Error())
-				return ctrl.Result{}, err
-			}
-			if r.hasExternalResources(blueprint) {
-				return ctrl.Result{RequeueAfter: 2 * time.Second}, errors.NewPlain("helm release uninstall is still in progress")
 			}
 			// remove the finalizer from the list and update it, because it needs to be deleted together with the object
 			ctrlutil.RemoveFinalizer(blueprint, finalizerName)
```

Comment:
The behavior of the algorithm changes here. If errors occur during the uninstall the blueprint does no longer wait for it to recover but just logs an error in the blueprint controller and removes the finalizer which means that the blueprint is good for deletion. I'm ok with this as otherwise the blueprint may be indefinitely stuck. 
Please just confirm that this is the wanted behavior.

Reply:
This is the desired behavior.